webpack.common.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = function (outputFilename) {
         'okta/jquery': SHARED_JS + '/util/jquery-wrapper',
         'okta/underscore': SHARED_JS + '/util/underscore-wrapper',
         'okta/handlebars': 'handlebars/dist/handlebars',
-        'okta/moment': EMPTY,
+        'okta/moment': 'moment/moment',
```

Comment:
@rchild-okta we need to add moment now, this probably will increase the size of the widget. Just FYI, unless you know a better way to add the time functionality that is needed)